cases/v41/const.py

```diff
@@ -41,91 +28,53 @@
-
-
-# test_he_install
-he_install = ['check_he_install']
 he_install_id = [18667, 18674, 18675, 18677]
 
-
-# test_he_install_bond
-he_install_bond = ['check_he_install_bond']
-he_install_bond_id = [18674]
-
-
-# test_he_install_bv
-he_install_bv = ['check_he_install_bv']
-he_install_bv_id = [18677]
-
-
-# test_he_install_vlan
-he_install_vlan = ['check_he_install_vlan']
-he_install_vlan_id = [18675]
-
-
-# test_he_install_non_default_port
-he_install_non_default_port = ['check_he_install_non_default']
-he_install_non_default_port_id = [18667]
-
-
-# test_he_install_redeploy
-he_install_redeploy = ['check_he_install_redploy']
-he_install_redeploy_id = [18686]
-
-
-# test_he_info
 he_info = [
     'check_engine_status_func', 'check_vm_status_func',
     'check_three_buttons_func', 'check_he_running_on_host_func',
-    'check_no_password_saved_func', 'check_add_additional_host', 
+    'check_no_password_saved_func', 'check_he_install_non_default', 
+    'check_he_install_redeploy', 'check_add_additional_host', 
     'check_put_local_maintenance', 'check_remove_from_maintenance', 
     'check_put_global_maintenance'
 ]
-he_info_id = [18669, 18670, 18671, 18672, 18685, 18668, 18678, 18679, 18680]
+he_info_id = [18669, 18670, 18671, 18672, 18685, 18667, 18686, 18668, 18678, 18679, 18680]
 
-
-# test_he_info_add_host
-he_info_add_host = [
-    'check_add_additional_host', 'check_put_local_maintenance', 
-    'check_remove_from_maintenance', 'check_put_global_maintenance'
-]
-he_info_add_host_id = [18668, 18678, 18679, 18680]
-
-
-# test_vm_registerd
-vm_registerd = [
+vm = [
     'check_running_vms_register_func', 'check_vdsm_func',
     'check_vm_login_logout_engine_func', 'check_vm_refresh_func',
-    'check_non_root_alert_func' 
-]
-vm_registerd_id = [18805, 18808, 18809, 18811, 18813]
-
-
-# test_vm_unregisterd
-vm_unregisterd = [
-    'check_running_vms_unregister_func', 
+    'check_non_root_alert_func', 'check_running_vms_unregister_func', 
     'check_vms_in_cluster_unregister_func' 
 ]
-vm_unregisterd_id = [18803, 18804]
+vm_id = [18805, 18808, 18809, 18811, 18813, 18803, 18804]
+
+deployment_cases_RHHI = [
+    'check_engine_lv_of_type_thick_and_volume_of_type_replicate',
+    'check_gluster_packages_presence_on_rhvh_node', 
+    'check_glusterfs_firewall_service_availability_with_default_firewallzone',
+    'check_cockpitui_should_be_reachable_for_the_user',
+    'check_option_to_start_with_gluster_deployment',
+    'check_saving_the_generated_gdeploy_config_file',
+    'check_cockpit_gdeploy_plugin_provides_redeploy_button',
+    'check_cleanup_of_gluster_setup_done',
+    'check_deployment_with_hostedengine_on_gluster',
+    'check_gluster_deployment_wizard',
+    'validate_host_deployment_tab',
+    'check_back_and_cancel_buttons_on_gdeploy_wizard',
+    'validate_arbiter_volume_creation',
+    'validate_packages_tab'
+]
+
+deployment_cases_RHHI_id =[138, 118, 119, 121, 122, 127, 129, 130, 132, 144, 145, 148, 147, 146]
```

Comment:
I won't add comment one by one, we would suggest to keep existing cases and id as original, and just add deployment_ cases_RHHI and deployment_cases_RHHI_id is much better.

Reply:
I am sorry, i did not quite understand what you meant here. Can you please elaborate?